kn/fotos/api.py

```diff
@@ -61,12 +62,18 @@ def _list(data, request):
     user = request.user if request.user.is_authenticated() else None
     return album_json(album, user)
 
-def _set_title(data, request):
+def _set_metadata(data, request):
     if 'title' not in data:
         return {'error': 'missing title attribute'}
     if not isinstance(data['title'], basestring):
         return {'error': 'title should be string'}
-    title = data['title']
+    title = data['title'] or None
```

Comment:
Ja.  Het kost iets meer ruimte, maar het is voor de gemiddelde Python programmeur niet verwarrend.